src/components/UserInfo.js

```diff
@@ -1,19 +1,36 @@
 export default class UserInfo {
-  constructor(userNameSelector, descriptionNameSelector) {
-    this._nameElement = document.querySelector(userNameSelector);
-    this._descriptionElement = document.querySelector(descriptionNameSelector);
+  #nameElement;
+  #descriptionElement;
+  #avatarElement;
+
+  constructor({ name, about, avatar, _id }) {
+    this._name = name;
+    this._about = about;
+    this._avatar = avatar;
+    this._id = _id;
+    this.#nameElement = document.querySelector(".profile__title");
+    this.#descriptionElement = document.querySelector(".profile__description");
+    this.#avatarElement = document.querySelector(".profile__image");
```

Comment:
According to the task `UserInfo` should accept 3 **selectors** to search 3 elements: 



```js
class UserInfo {
  constructor({ nameSelector, jobSelector, avatarSelector }) {
    this._nameElement = document.querySelector(nameSelector);
    this.jobElement = document.querySelector(jobSelector);
    this._avatarElement = document.querySelector(avatarSelector);
  }


```